supporter-product-data/src/test/scala/com/gu/model/dynamo/SerialisationSpec.scala

```diff
@@ -0,0 +1,28 @@
+package com.gu.model.dynamo
+
+import com.gu.Fixtures
+import kantan.csv.ops.toCsvInputOps
+import kantan.csv.rfc
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+class SerialisationSpec extends AnyFlatSpec with Matchers {
+  SupporterRatePlanItem.getClass.getSimpleName should "deserialise correctly" in {
```

Comment:
clever way to get the test name from the source code, nice bonus for refactoring later!